Formula/nzbget.rb

```diff
@@ -52,6 +52,11 @@ def plist; <<~EOS
     <dict>
       <key>Label</key>
       <string>#{plist_name}</string>
+      <key>EnvironmentVariables</key>
+      <dict>
+        <key>PATH</key>
+        <string>/usr/local/bin/:/usr/bin:$PATH</string>
```

Comment:
- Isn't `/usr/bin` in `PATH` by default?
- Use `#{HOMEBREW_PREFIX}/bin` for Homebrew path

Reply:
I copied this from another formula. I've made the requested changes.